packages/design-system/src/components/ChoiceList/Choice.jsx

```diff
@@ -59,11 +67,11 @@ export class Choice extends React.PureComponent {
 
     if (!this.isControlled) {
       this.setState({ checked: evt.target.checked });
-    }
```

Comment:
So the event emitter should only fire for uncontrolled Choices right?

Reply:
Wasn't the MCT example controlled?

Reply:
MCT had controlled radio button on the top level. The issue was for when MCT uses the uncontrolled radio button further done the chain.

Reply:
and when you reproduced it, you were using uncontrolled right?

Reply:
Yes. I tested will controlled & uncontrolled, radio & check buttons. The issue only occurs for uncontrolled radio buttons because of the use of emitter and not cleaning out the emitter on the unmount of the component. Hence, the component is referencing this leftover ref pointing to null value.